modules/nf-core/homer/findpeaks/tests/main.nf.test

```diff
@@ -69,6 +83,32 @@ nextflow_process {
             )
         }
     }
-    // TODO Test uniqmap. Might need to make a small uniqmap but that could take a day or two.
-    // See https://github.com/Functional-Genomics-Lab/homer-uniqmap-nf if you're feeling adventurous!
+
+    test("sarscov2 - tagdir - uniqmap") {
+
+        tag "uniqmap"
+        options "-stub"
```

Comment:
```suggestion
```
This will help with the empty tests.

Reply:
I caught that one eventually, just hadn't pushed the commit up 😆 https://github.com/nf-core/modules/commit/5d8544b062dcaf9419b97cbab5e16aa5bdaa7bc2